cos-fleetshard-sync/src/main/java/org/bf2/cos/fleetshard/sync/FleetShardSyncConfig.java

```diff
@@ -118,6 +118,15 @@ interface Resources {
         @WithDefault("15s")
         @WithConverter(DurationConverter.class)
         Duration updateInterval();
+
+        /**
+         * Determine how often the synchronizer should perform house keeping tasks.
+         *
+         * @return the housekeeping interval
+         */
+        @WithDefault("30s")
```

Comment:
could be too much to wait for addon deletion?

Reply:
It could be.
I guess we need to run some real tests and fine a reasonable number.